### PR TITLE
Update TrustDice.Win.fc

### PR DESCRIPTION
**Pull request by aviramdayan**
change-site-all-elements-new

### CODE REVIEW

File: TrustDice.Win/TrustDice.Win.fc [hunk 1]
```diff
@@ -219,17 +208,8 @@ public class TrustDice : FaucetScript
...
-             var MinVAR = ElementByXPath("//div[contains(@class,'faucet_countdown')]/span[3]") ;
-             var SecVAR = ElementByXPath("//div[contains(@class,'faucet_countdown')]/span[5]") ; 
-			 
-             int hour,min,sec;
-             hour = Convert.ToInt32(HourVAR.Text)*3600;
-             min = Convert.ToInt32(MinVAR.Text)*60;
-             sec = Convert.ToInt32(SecVAR.Text);
-             return hour+min+sec;
 
+             return 360*60;
```
Comment (faucetcollector) [inline TrustDice.Win/TrustDice.Win.fc]: no more custom timer?

File: TrustDice.Win/TrustDice.Win.fc [hunk 2]
```diff
@@ -391,26 +371,15 @@ public class TrustDice : FaucetScript
 
              Wait(2);
 			 
-		     var ButtonFaucet = ElementByXPath("//div[contains(@class,'faucet_claim-panel__GGcxk faucet_BTC')]/button"); //("//button[contains(text(),'Claim')] | //button[contains(text(),'claim')]")
+		     var ButtonFaucet = ElementByXPath("//div[2]/div[2]/div/button[contains(@class,'MuiButton-outlined MuiButton-outlinedSecondary')]");
```
Comment (faucetcollector) [inline TrustDice.Win/TrustDice.Win.fc]: this one is very likely to break as soon as they push an update to their website

File: TrustDice.Win/TrustDice.Win.fc (hunk 1 above)
Comment (aviramdayan) [reply TrustDice.Win/TrustDice.Win.fc]: there is but very tough to solve/get this element
image
the `jss` set random number Always
I tried to do that,
But I'm not coming to it
The whole site is very difficult ..

File: TrustDice.Win/TrustDice.Win.fc (hunk 2 above)
Comment (aviramdayan) [reply TrustDice.Win/TrustDice.Win.fc]: This is the only option that is possible, I tried to find a lot of shortcuts ..

Comment (faucetcollector) [reply TrustDice.Win/TrustDice.Win.fc]: feel free to merge the PR, I did approve it

File: TrustDice.Win/TrustDice.Win.fc (hunk 1 above)
Comment (aviramdayan) [reply TrustDice.Win/TrustDice.Win.fc]: i wait a little, if you can get the element

Comment (aviramdayan) [reply TrustDice.Win/TrustDice.Win.fc]: this is the only option
("//div[2 and contains(@class,'main-bg')]/div[1]/div/div/div[1]/div/div/p[1]")
("//div[2 and contains(@class,'main-bg')]/div[1]/div/div/div[1]/div/div/p[2]") 
("//div[2 and contains(@class,'main-bg')]/div[1]/div/div/div[1]/div/div/p[3]") 
i send new commit

**Comment by Aragorn443** [comment]
Hi, I am just a beginner with scripts, and recently found this repository on github. Can someone please guide me on how to actually use one of these scripts?

**Comment by aviramdayan** [comment]
> Hi, I am just a beginner with coding, and recently found this repository on github. Can someone please guide me on how to actually use one of these scripts?

Hello, the scripts are written in language -
  JS / C # .NET / XPATH
You can learn syntax
FC Documentation
https://faucetcollector.com/docs/index.html
You can start studying XPATH here:
https://www.guru99.com/xpath-selenium.html
C # .NET API
https://selenium.dev/selenium/docs/api/dotnet/#
Boolean logical operators (C # reference)
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/boolean-logical-operators#logical-or-operator-
JS
https://www.learn-js.org/
And there are many more, but start from here,
If you have a question, we're here to help

**Pull request merged**